app/controllers/concerns/restricted_token_magic_links_management.rb

```diff
@@ -1,4 +1,10 @@
 module RestrictedTokenMagicLinksManagement
+  def new
+    @token = Token.find(params[:id])
+
+    render 'shared/tokens/transfer'
+  end
```

Comment:
Ce serait pas mieux d'apeller la route transfer du coup, surtout que tu name la route `token_transfer` ?

Reply:
J'aurais fait un concern/controller `TransferToken` à part perso.

Reply:
Je pense que je vais faire un controller comme dit @skelz0r

Reply:
En vrai je pense que la méthode a sa place dans le controller/concern `RestrictedTokenMagicLinks`, au pire c'est ce module entier qu'il faudrait renommer en `TransferToken`

du coup je laisse comme ça pour le moment, si on trouve ça pas assez propre on fera sa propre issue

Reply:
Si faut juste rename do it, j'ai pas assez de connaissance métier sur la chose pour m'exprimer (modulo que `RestrictedTokenMagicLinks` c'est pas super clair).

Reply:
je vais quand même bouger la vue parce qu'elle devrait probablement être ailleurs que dans `token`,

Reply:
Ok je vais rename :+1:

Reply:
tant qu'a faire je vais renommer "PublicTokenMagicLink" en "MagicLink"